app/client/src/components/StartButton.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div class="start-analysis">
+    <button v-if='!analysisStatus.submitted && allSketched' class="btn btn-block btn-standard"
+    @click='runPoppunk();setStatus({ task: "submitted", data: "submitted"})'>
+      Start Analysis</button>
+    <button v-else class="btn btn-block btn-disabled">
+      Start Analysis</button>
```

Comment:
You should be able to avoid some repetition here if you just want to disable the button under that condition.
```suggestion
    <button :class="(!analysisStatus.submitted && allSketched) ? '' : 'disabled'" class="btn btn-block btn-standard"
    @click='runPoppunk();setStatus({ task: "submitted", data: "submitted"})'>
      Start Analysis</button>
```